sandboxed_containers/sandboxed-containers-4.11-release-notes.adoc

```diff
@@ -26,7 +26,78 @@ This product is fully supported and enabled by default as of {product-title} 4.1
++
+If the mounted files or directories do not have specific SELinux requirements on the host, you can use local persistent volumes as an alternative. Files are automatically relabeled to `container_file_t`, following SELinux policy for container runtimes. See xref:../storage/persistent_storage/persistent-storage-local.adoc[Persistent storage using local volumes] for more information.
++
+Automatic relabeling is not an option when mounted files or directories are expected to have specific SELinux labels on the host. Instead, you can set custom SELinux rules on the host in order to allow the `virtiofsd` daemon to access these specific labels. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1904609[*BZ#1904609*])
+
+* The {sandboxed-containers-operator} pods that use container CPU resource limits to increase the number of available CPUs for the pod might receive fewer CPUs than requested. If the functionality is available inside the container, you can diagnose CPU resources by using `oc rsh <pod>` and running the `lscpu` command.
++
+[source,terminal]
+----
+$ lscpu
+----
++
+.Example output
++
+[source,terminal]
+----
+CPU(s):                          16
+On-line CPU(s) list:             0-12,14,15
+Off-line CPU(s) list:            13
+----
++
+The list of available offline CPUs will likely change from run to run in an unpredictable manner.
++
+As a workaround, you can use a pod annotation to request additional CPUs rather than setting a CPU limit. The method of allocating processors is different, and CPUs requested by means of pod annotation are not affected by this issue. Rather than setting a CPU limit, the following `annotation` must be added to the metadata of the pod:
++
+[source,yaml]
+----
+metadata:
+  annotations:
+    io.katacontainers.config.hypervisor.default_vcpus: "16"
+----
++
+(link:https://issues.redhat.com/browse/KATA-1376[*KATA-1376*])
+
+* The progress of the runtime installation is shown in the `status` section of the `kataConfig` CR. However, the progress is not shown if all of the following conditions are true:
+
+** The cluster has a machine config pool `worker` without any members (`machinecount=0`).
+** No `kataConfigPoolSelector` is specified to select nodes for installation.
+
++
+In this case, the installation starts on the master nodes because the Operator assumes it is a converged cluster where nodes have both master and worker roles. The `status` section of the `kataConfig` CR is not updated during the installation. (link:https://issues.redhat.com/browse/KATA-1017[*KATA-1017*])
+
+* When using older versions of the buildah tool in {sandboxed-containers-first}, the build fails with the following error:
++
+[source,text]
+----
+process exited with error: fork/exec /bin/sh: no such file or directorysubprocess exited with status 1
```

Comment:
Is "directorysubprocess" one word in the error output? My assumption is that it is, but I thought I'd mention it here just to check.

Reply:
This was the output supplied by engineering and reviewed by QE, but I can ask again.

Reply:
It looks to me that there should be a newline between `directory` and `subprocess`.

Reply:
@MirzWeiss can you please add the new line between the two words? Thanks!

Reply:
Done.